disnake/ui/action_row.py

```diff
@@ -185,10 +201,12 @@ def children(self) -> Sequence[UIComponentT]:
     def width(self) -> int:
         return sum(child.width for child in self._children)
 
-    def append_item(self, item: UIComponentT) -> None:
+    def append_item(self, item: UIComponentT) -> Self:
         """Append a component to the action row. The component's type must match that
         of the action row.
 
+        This function returns the class instance to allow for fluent-style chaining.
```

Comment:
In addition to this, these docstrings should also have a `Returns` section

Reply:
Hm, on second thought, the `Embed` methods that allow similar chaining don't have that either. Disregard my comment lol

Reply:
Could you please re-approve, then?